.packit.yaml

```diff
@@ -0,0 +1,36 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: qpc.spec
+
+upstream_project_url: https://github.com/quipucords/qpc
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets: &targets_list
+      # we officially support rhel-8 and rhel-9 downstream.
+      - rhel-8
+      - rhel-9
+      # centos-stream is like a preview of upcoming rhel.
+      - centos-stream-8
+      - centos-stream-9
+      # fedora is like an even farther future preview.
+      - fedora-37
```

Comment:
I get we need fedora 38 for Jenkins, but why fedora 37? 🪓

Reply:
Good point! I only included that because it was part of Packit's `fedora-all` group alias that targets all current supported versions. 🪓-ing!